wgpu/src/util/belt.rs

```diff
@@ -120,7 +120,7 @@ impl StagingBelt {
         size: BufferSize,
         alignment: BufferSize,
         device: &Device,
-    ) -> (BufferViewMut<'_>, &Buffer, BufferAddress) {
+    ) -> (BufferViewMut<'_>, BufferSlice<'_>) {
```

Comment:
Would it make sense to just return the buffer slice?

Reply:
Indeed, that is a possibility; I went for the minimal change instead.

* For: fewer parts, API elegance (not returning a tuple is nice).
* Against: exposing more of how the belt works and requiring the caller to deal with mapping; cannot *add* anything intermediate without breaking API (but it's doubtful any such thing would happen, let alone without other breaking changes).

Given that we both thought of it, probably we should? What do you think of these thoughts?

Reply:
It was just a thought, I don't feel very strongly so I'll deffer to you on this. I am not too worried about the breaking change (or rather, I think that now is a better time to experiment and change our minds than later when wgpu gets more and more API-stable).
It looks to me like it might be more convenient with the slice instead of the tuple, but I don't have much first hand experience with the staging belt.

Reply:
I've changed it to just return the slice. I also updated the documentation, which I forgot to do entirely.